### PR TITLE
node status

### PR DESCRIPTION
**Pull request by Heisenberg1337**
Added Online and Included properties for the nodes. Online is node status based on ping, Included is node status based on whether this node must be unavailable for reservation for some reason or not.

### CODE REVIEW

File: pkg/mapr/DBUtils.go [hunk 1]
```diff
@@ -63,8 +62,16 @@ func GetAvailableNodes(clusterID string, operatingSystem string, osVersion strin
...
 	// query for nodes where the ExpiresAt field has already passed
-	queryStr := fmt.Sprintf(`{"$where":{"$and":[{"$matches":{"NodeObj.OperatingSystem.Name":"(?i)%s"}},{"$lt":{"ExpiresAT": "%s"}},{"$matches":{"NodeObj.OperatingSystem.Version":"%s"}}] }}`,
-		operatingSystem, time.Now().Format(time.RFC3339), osVersion)
+	queryStr := fmt.Sprintf(`
+{"$where":{"$and":[
+	{"$matches":{"NodeObj.OperatingSystem.Name":"(?i)%s"}},
+	{"$lt":{"ExpiresAT": "%s"}},
+	{"$matches":{"NodeObj.OperatingSystem.Version":"%s"}},
+	{"$eq":{"Online": %t}},
+	{"$eq":{"Included":%t}}
```
Comment (sogwiz) [inline pkg/mapr/DBUtils.go]: I'm guessing we'll re-do this since we just discussed the idea of removing the "Included" field

**Review by sogwiz — CHANGES_REQUESTED** [review]
Now that we clarified on the requirements today, I'm thinking that we'll need to revise this PR to remove the included field, right?

File: pkg/mapr/DBUtils.go (hunk 1 above)
Comment (Heisenberg1337) [reply pkg/mapr/DBUtils.go]: Done

File: pkg/models/Models.go [hunk 2]
```diff
@@ -19,6 +19,8 @@ type NodeDBJson struct {
 	NodeObj   Node
 	ClusterID string
 	ExpiresAT string
+	Offline   bool
```
Comment (sogwiz) [inline pkg/models/Models.go]: based off our decision here https://github.com/sogwiz/vmrenter/pull/7/files#diff-bf1eee6244c5481b1babe4ebe76f26d0R15 , this should be "isOffline"

**Review by sogwiz — CHANGES_REQUESTED** [review]
please see comment.

File: pkg/models/Models.go (hunk 2 above)
Comment (Heisenberg1337) [reply pkg/models/Models.go]: fixed

**Review by sogwiz — APPROVED** [review]
ship it!

**Pull request merged**